packages/router/src/adapters/subgraph/subgraph.ts

```diff
@@ -116,19 +116,18 @@ export const getActiveTransactions = async (_requestContext?: RequestContext): P
         // update synced status
         await setSyncRecord(chainId, requestContext);
 
-        // get all receiver expired txs
-        const allReceiverExpired = await sdk.request<GetReceiverTransactionsQuery>((client) =>
+        // get all receiver prepared txs
+        const allReceiverPrepared = await sdk.request<GetReceiverTransactionsQuery>((client) =>
           client.GetReceiverTransactions({
             routerId: routerAddress.toLowerCase(),
             receivingChainId: chainId,
             status: SdkTransactionStatus.Prepared,
             expiry_lt: Math.floor(Date.now() / 1000),
           }),
         );
-        if ((allReceiverExpired.router?.transactions.length ?? 0) > 0) {
-          logger.debug("Got receiver expired", requestContext, methodContext, {
+        if ((allReceiverPrepared.router?.transactions.length ?? 0) > 0) {
+          logger.debug("Got receiver prepared", requestContext, methodContext, {
```

Comment:
are we properly handling cases with receiver expired? `ReceiverExpired` means the router has to take the action of cancelling that tx. This query seems to get the expired + prepared txs only, so maybe the variable is misnamed?